xml/yast2_gui.xml

```diff
@@ -13,14 +13,108 @@
+  &yast;. Written in Ruby, &yast; features an extensible architecture that
+  makes it possible to add new functionality via modules.
  </para>
- <sect1 xml:id="sec-yast-gui-key-combination">
-  <title>Advanced Key Combinations</title>
+ <para>
+  Additional information about &yast; is available on the project's official
+  website at <link xlink:href="https://yast.opensuse.org/"/>.
+ </para>
+ <sect1 xml:id="sec-yast2-refs">
+  <title>&yast; Topics</title>
+  <para>
+   Detailed information on using specific &yast; modules and functionality is
+   covered throughout the documentation.
+  </para>
+  <para>
+   <xref linkend="cha-onlineupdate-you" />
+  </para>
+  <para>
+   <phrase os="sles;osuse"><xref linkend="sec-apache2-configuration-yast" /></phrase>
+  </para>
+  <para>
+   <xref linkend="sec-grub2-yast2-config" />
+  </para>
+  <para>
+   <xref linkend="sec-network-yast" />
+  </para>
+  <para>
+   <phrase os="sles;osuse"><xref linkend="cha-ftp" /></phrase>
+  </para>
+  <para>
+   <xref linkend="cha-yast-serviceman" />
+  </para>
+  <para>
+   <xref linkend="cha-yast-lang" />
+  </para>
+  <para>
+   <xref linkend="sec-yast-hw-print" />
+  </para>
+  <para>
+   <phrase os="sled;osuse"><xref linkend="sec-yast-hw-scan" /></phrase>
+  </para>
+  <para>
+   <xref linkend="sec-yast-hw-sound" />
+  </para>
+  <para>
+   <xref linkend="cha-add-ons" />
+  </para>
+  <para>
+   <xref linkend="sec-yast-software-instsource" />
+  </para>
+  <para>
+   <xref linkend="cha-yast-software" />
+  </para>
+  <para>
+   <xref linkend="cha-yast-userman" />
+  </para>
+  <para>
+   <xref linkend="cha-yast-hw" />
+  </para>
+  <para>
+   <xref linkend="sec-yast-hw-keym" />
+  </para>
+  <para>
+   <xref linkend="cha-apparmor-yast" />
+  </para>
+  <para>
+   <xref linkend="sec-security-ldap-yast-client" />
+  </para>
+  <para>
+   <xref linkend="sec-security-kerberos-yast-client" />
+  </para>
+  <para>
+  <phrase os="sles;osuse"><xref linkend="sec-security-yastvpn" /></phrase>
+  </para>
+  <para>
+   <xref linkend="cha-security-yast-ca" />
+  </para>
+  <para>
+   <xref linkend="cha-security-yast-security" />
+  </para>
+ </sect1>
+ 
+ <sect1 xml:id="sec-yast2-gui">
+  <title>&yast; Interface Overview</title>
+  <para>
+   &yast; has two graphical interfaces: one for use with graphical desktop
+   environments like KDE and GNOME, and an ncurses-based pseudo-graphical
+   interface for use on systems without an X server.
+  </para>
+  <para>
+   In the graphical version of &yast;, all modules in &yast; are grouped by
+   category, and the navigation sidebar allows you to quickly access modules in
+   the desired category. The search field at the top makes it possible to find
+   modules by their names. To find a specific module, enter its name into the
+   search field, and you should see the modules that match the entered string
+   as you type.
+  </para>
+   <sect2 xml:id="sec-yast-gui-key-combination">
+  <title>Useful Key Combinations</title>
```

Comment:
Maybe it's worth mentioning that the key combos only work for graphical YaST? As this is a subsection of sec-yast2-gui now, it is not entirely clear that it belongs to graphical YaST only.

Also, this is a 'lonely' sect2 (there is no other corresponding sect2). Could it be promoted to a sect1 instead?